backend/src/util/mod.rs

```diff
@@ -1 +1,146 @@
+    Lu,
+    /// This meal contains almonds.
+    Ma,
+    /// This meal contains milk / lactose.
+    ML,
+    /// This meal contains brazil nuts.
+    Pa,
+    /// This meal contains pecans.
+    Pe,
+    /// This meal contains pistachios.
+    Pi,
+    /// This meal contains macadamia nuts.
+    Qu,
+    /// This meal contains rye and rye gluten.
+    Ro,
+    /// This meal contains sesame.
+    Sa,
+    /// This meal contains celery.
+    Se,
+    /// This meal contains sulphite.
+    Sf,
+    /// This meal contains mustard.
+    Sn,
+    /// This meal contains soya.
+    So,
+    /// This meal contains walnuts.
+    Wa,
+    /// This meal contains wheat and wheat gluten.
+    We,
+    /// This meal contains molluscs.
+    Wt,
+}
+
+/// This enum lists every possible additive a meal can have.
+pub enum Additive {
+    /// This meal contains colorants.
+    Colorant,
+    /// This meal contains preserving agents.
+    PreservingAgents,
+    /// This meal contains antioxidant agents.
+    AntioxidantAgents,
+    /// This meal contains flavour enhancers.
+    FlavourEnhancer,
+    /// This meal contains phosphate.
+    Phosphate,
+    /// This meals surface is waxed.
+    SurfaceWaxed,
+    /// This meals contains sulphir.
+    Sulphur,
+    /// This meals contains artificially blackened olives.
+    ArtificiallyBlackenedOlives,
+    /// This meals contains sweetener.
+    Sweetener,
+    /// This meals can be laxative if overused.
+    LaxativeIfOverused,
+    /// This meals contains phenylalanine.
+    Phenylalanine,
+    /// This meals can contain alcohol.
+    Alcohol,
+    /// This meals contains pressed meet.
+    PressedMeet,
+    /// This meals is glazed with cacao.
+    GlazingWithCacao,
+    /// This meals contains pressed fish.
+    PressedFish,
+}
+
+/// This enum lists all the types a meal can be of.
+pub enum MealType {
+    /// This meal is vegan.
+    Vegan,
+    /// This meal is vegetarian.
+    Vegetarian,
+    /// This meal contains beef.
+    Beef,
+    /// This meal contains beef from regional appropriate animal husbandry.
+    BeefAw,
+    /// This meal contains pork.
+    Pork,
+    /// This meal contains pork from regional appropriate animal husbandry.
+    PorkAw,
+    /// This meal contains fish.
+    Fish,
+    /// It is unknown whether this meal contains any meat or not.
+    Unknown,
+}
+
+/// This enum lists all the predetermined reasons a image can be reported for.
+pub enum ReportReason {
+    /// This picture shows offensive content.
+    Offensive,
+    /// This picture is an advert.
+    Advert,
+    /// This picture does not show a meal.
+    NoMeal,
+    /// This picture shows the wrong meal.
+    WrongMeal,
+    /// This picture violates my rights.
+    ViolatesRights,
+    /// This picture should be removed for some other reason.
+    Other,
+}
+
+/// Date type used in multiple paces.
+pub type Date = NaiveDate;
+
+/// This struct contains all price classes
+pub struct Price {
```

Comment:
Maybe specify in docs that prices are in (euro) cents?

Reply:
maybe yes